### PR TITLE
# 161249984 Fix View accomodation details

### PR DESCRIPTION
**Pull request by loicemeyo**
#### What does this PR do?
- Fixes Trip Details to so that the accommodation field to display null when in case of no bed space

#### Screenshots (if appropriate)
image

#### What are the relevant pivotal tracker stories?
[161249984](161249984)

### CODE REVIEW

File: src/components/RequestsModal/TripDetails/index.jsx [hunk 1]
```diff
@@ -52,9 +52,11 @@ export default class TripDetails extends PureComponent {
       origin,
       tripType
     } = tripDetails;
-    const {rooms:{roomName, guestHouses:{houseName}}, bedName } = beds;
-    const accomodationdetails = `${houseName}, ${roomName}, ${bedName}`;
-   
+    let accomodationdetails='';
```
Comment (daddychukz) [inline src/components/RequestsModal/TripDetails/index.jsx]: change this to `camelCase`

File: src/components/RequestsModal/TripDetails/index.jsx [hunk 2]
```diff
@@ -52,9 +52,11 @@ export default class TripDetails extends PureComponent {
...
+    if(beds){
+      const {rooms:{roomName, guestHouses:{houseName}}, bedName } = beds;
+      const accomodationdetails = `${houseName}, ${roomName}, ${bedName}`;
```
Comment (daddychukz) [inline src/components/RequestsModal/TripDetails/index.jsx]: you have already declared `accomodationdetails` in the upper scope, no need making it a constant again. Remove the `const`

**Comment by loicemeyo** [comment]
@daddychukz Thank you. I've implemented feedback

**Comment by loicemeyo** [comment]
@samuelkubai I've updated the PR

**Pull request merged**